package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "material-ui-json-schema-viewer",
+  "version": "1.0.0",
+  "main": "index.js",
```

Comment:
When we run `yarn build`, I think this will output the build inside the `build` directory. The main file would probably have to be `"main": "build/index.js"` in that case.

Reply:
Updated!